### PR TITLE
feat(api, types): add transfer_id to withdrawals

### PR DESCRIPTION
**Pull request by Samuel-Sendzul**
Add tranfer_id to Withdrawal and GetWithdrawalResponse to align with the Luno API.

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -898,6 +898,9 @@ type GetWithdrawalResponse struct {
 	// Status
 	Status Status `json:"status"`
 
+	// Transfer ID if the corresponding transfer if it has been initiated
```
Comment (pierredup) [inline api.go:901]:
```suggestion
	// Transfer ID of the corresponding transfer if it has been initiated
```

File: types.go [hunk 2]
```diff
@@ -557,6 +557,9 @@ type Withdrawal struct {
 	// Status
 	Status Status `json:"status"`
 
+	// Transfer ID if the corresponding transfer if it has been initiated
```
Comment (pierredup) [inline types.go:560]:
```suggestion
	// Transfer ID of the corresponding transfer if it has been initiated
```

**Comment by echarrod** [comment]
Thanks very much for your contribution @samrover2809. 

We actually generate this file automatically with a generator, so we’ll make this change independent of this MR, but will work on doing that now, thanks for the prompt

**Comment by echarrod** [comment]
Being added in https://github.com/luno/luno-go/pull/122, I'll close this one now 👍

**Pull request closed without merging**